README.md

```diff
@@ -2,39 +2,42 @@
 
 A mobile optimized sliding navigation plugin.
 
-## Dependencies
+[![NPM](https://nodei.co/npm/navitron.png?downloads=true)](https://nodei.co/npm/navitron/)
 
-* [Zepto](http://zeptojs.com/)
-* [Mobify's fork of Velocity.js](http://github.com/mobify/velocity)
-* [Plugin](http://github.com/mobify/plugin)
+## Demo
 
+You can find a simple demo inside the `examples` folder in the repo. Run `grunt examples` to see it in Chrome (mobile device emulation is recommended).
 
-### Velocity
+## Dependencies
```

Comment:
Does it make sense to list dependencies here? I just think about keeping this in sync with the real source of truth: `package.json`.